cli/e2e.js

```diff
@@ -206,9 +207,17 @@ function spawnBuild(argv, skyPagesConfig, webpack) {
  * @name e2e
  */
 function e2e(argv, skyPagesConfig, webpack) {
+  const specsPath = path.resolve(process.cwd(), 'e2e/**/*.spec.ts */');
```

Comment:
The glob pattern here has an extra trailing ` */`.  I forgot the `e2e-` to the pattern in my example.

The final glob pattern should be `e2e/**/*.e2e-spec.ts`